test/e2e_new/conformance_test.go

```diff
@@ -99,3 +105,56 @@ var _ = Describe("conformance tests", func() {
 	})
 
 })
+
+func prepConformanceConfig(artifactsDir string, e2eConfig *clusterctl.E2EConfig) error {
+	for i, p := range e2eConfig.Providers {
+		if p.Name != "aws" {
+			continue
+		}
+		e2eConfig.Providers[i].Files = append(p.Files, clusterctl.Files{
+			SourcePath: path.Join(artifactFolder, "/templates/cluster-template-conformance-ci-artifacts.yaml"),
+			TargetName: "cluster-template-conformance-ci-artifacts.yaml",
```

Comment:
Is there no way currently to trigger a conformance test against a released version of Kubernetes? It looks like everything is hard coded towards running against ci artifacts.

Reply:
We always generate the conformance template, but https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/test/e2e_new/conformance_test.go#L59 determines whether or not it actually gets used.